keyboards/marcopad/keyboard.json

```diff
@@ -0,0 +1,58 @@
+{
+	"manufacturer": "pelegrini.ca",
+	"keyboard_name": "marcopad",
+	"maintainer": "Marco Pelegrini",
+	"development_board": "promicro_rp2040",
```

Comment:
Could `development_board` be replaced by `processor` and `bootloader'? As the WaveShare RP2040 Zero has a different physical footprint and different peripheral configuration from the SparkFun Pro Micro RP2040 🙈